main.go

```diff
@@ -23,38 +40,78 @@ var postgresFunctions = []runtime.Transform{
 	},
 }
 
+var (
+	Network  = "unix"
+	LogLevel = 1
+)
+
+type server struct {
+	pb.UnimplementedContainerizedFunctionRunnerServiceServer
+	logger logr.Logger
+}
+
+func (s *server) RunFunction(ctx context.Context, in *pb.RunFunctionRequest) (*pb.RunFunctionResponse, error) {
+	ctx = logr.NewContext(ctx, s.logger)
+	_ = runtime.LogMetadata(ctx, AI)
```

Comment:
I probably wouldn't log the metadata on each an every request, but only on startup